backend/src/envPlatform/schema.bal

```diff
@@ -0,0 +1,94 @@
+
+type TreeRemovalApplication record { 
+     int numberOfVersions;
+     int versionNumber;
+     string applicationId;
+     string title;
+     Date applicationCreatedDate;
+     Date removalDate;
+     string reason;
+     string 'type;
```

Comment:
```suggestion
     string type;
```
If the apostrophe was put because type is recognized as an internal variable, let's change this to **applicationType**